src/lib.rs

```diff
@@ -79,35 +95,44 @@ fn assign_color_to_ai(str: String, human: u8) -> Goban {
 
     if human == WHITE {
         Goban::new(player, enemy)
-    }
-    else {
-        println!("Goban after color assign : \n{:?}", Goban::new(enemy, player)); //to remove
+    } else {
+        println!(
+            "Goban after color assign : \n{:?}",
+            Goban::new(enemy, player)
+        ); //to remove
         Goban::new(enemy, player)
     }
 }
 
 fn launch_ai(input: Goban, player_captures: u8, opponent_captures: u8) -> (u32, u32) {
     let mut algorithm = Algorithm::new();
-    algorithm.update_initial_state(input, *input.get_enemy(), player_captures, opponent_captures);
+    algorithm.update_initial_state(
+        input,
+        *input.get_enemy(),
+        player_captures,
+        opponent_captures,
+    );
     let ret = algorithm.get_next_move(DEPTH).unwrap();
-    println!("RET = \n{:?}", ret);
+    // println!("RET = \n{:?}", ret);
 
     println!("Get win coord inside launch_ai : "); //to remove
-    println!("{:?}", get_win_coord(*input.get_player(), *ret.get_item().get_player())); //to remove
+    println!(
+        "{:?}",
+        get_win_coord(*input.get_player(), *ret.get_item().get_player())
+    ); //to remove
     get_win_coord(*input.get_player(), *ret.get_item().get_player())
 }
 
 fn get_win_coord(previous: BitBoard, current: BitBoard) -> (u32, u32) {
     let pos = previous ^ current;
-    println!("PREV:\n{}\nCUR:\n{}", previous, current);
-    println!("UNIQUE POS:\n{}", pos);
+    // println!("PREV:\n{}\nCUR:\n{}", previous, current);
+    // println!("UNIQUE POS:\n{}", pos);
 
-    let i : u32 = *pos.get_bit_indexes().last().unwrap() as u32;
+    let i: u32 = *pos.get_bit_indexes().last().unwrap() as u32;
     println!("I is = {}, coord = {:?}", i, (i / 20, i % 20));
     // println!("{}", pos);
     (i / 20, i % 20)
 }
 
 #[cfg(test)]
-mod tests {
-}
+mod tests {}
```

Comment:
It's probably better to remove it completely as long as there are no unit tests for this mod.